readthedocs/projects/constants.py

```diff
@@ -47,6 +47,15 @@
     MEDIA_TYPE_JSON,
 )
 
+# Map media types to their know extensions.
+# This is used for validating and uploading artifacts.
+MEDIA_TYPES_EXTENSIONS = {
+    MEDIA_TYPE_PDF: ("pdf",),
+    MEDIA_TYPE_EPUB: ("epub",),
+    MEDIA_TYPE_HTMLZIP: ("zip",),
+    MEDIA_TYPE_JSON: ("json",),
+}
```

Comment:
Are we using a tuple in the value here because we want to have multiple extensions per each media type?